src/main/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ endif()
 
 add_executable(cvc5-bin driver_unified.cpp main.cpp $<TARGET_OBJECTS:main>)
 target_compile_definitions(cvc5-bin PRIVATE -D__BUILDING_CVC5DRIVER -Dcvc5_obj_EXPORTS)
+target_link_options(cvc5-bin PRIVATE ${WASM_FLAGS})
```

Comment:
Instead of modifying the linker options for only this binary, let's do it globally for all binaries like we do for other linker options: https://github.com/cvc5/cvc5/blob/6791c74f34495d6bc3f2f097a281d82835bf9808/CMakeLists.txt#L216